10-kubernetes-inputs.tf

```diff
@@ -80,4 +80,9 @@ variable "control_resource_group" {
 variable "enable_automatic_channel_upgrade_patch" {
   description = "When set to true automatic patch updates will be enabled on the cluster"
   default     = false
+}
+
+variable "enable_recommended_max_surge" {
```

Comment:
is there a reason to flag this? shouldn't we just set a reasonable default and allow people to override it? i.e. they could override back to 1 to disable it

Reply:
Yeah I was going to see how it went with the CFT upgrades then probably just remove it. Should I just remove the toggle now @timja?

Reply:
I would yeah